ms_access2pg_dump.bas

```diff
@@ -239,12 +242,20 @@ Sub MSAccessForeignKeys2PGDump(out_file As String, Optional fileAppend As Boolea
       line = "alter table " & strQuote(Relation.ForeignTable, Chr$(34)) _
         & " add foreign key (" & Join(foreign_cols, ", ") & ")" _
         & " references " & strQuote(Relation.Table, Chr$(34)) & " (" & Join(cols, ", ") & ")"
-      If (Relation.Attributes & dbRelationUpdateCascade) <> 0 Then
-        line = line + " on update cascade"
+      If tempRelationAttributes >= dbRelationRight Then
```

Comment:
Hi there I've made fix to our code that corrects a typo.
We should have been using bitwise operators to test for dbRelationUpdateCascade etc.

I think with the change made in https://github.com/qcode-software/ms_access2pg_dump/pull/3/files this should now work as expected.